lib/common/Memory/SmallFinalizableHeapBlock.h

```diff
@@ -103,7 +103,11 @@ class SmallFinalizableHeapBlockT : public SmallNormalHeapBlockT<TBlockAttributes
 
     virtual bool FindHeapObject(void* objectAddress, Recycler * recycler, FindHeapObjectFlags flags, RecyclerHeapObjectInfo& heapObject) override
     {
-        return FindHeapObjectImpl<SmallFinalizableHeapBlockT>(objectAddress, recycler, flags, heapObject);
+#if defined(_MSC_VER)
+		return FindHeapObjectImpl<SmallFinalizableHeapBlockT>(objectAddress, recycler, flags, heapObject);
+#else
+		return this->template FindHeapObjectImpl<Base>(objectAddress, recycler, flags, heapObject);
```

Comment:
I'm open to suggestions on a better way to do this, but this is the simplest one that actually works